aimmo-game/simulation/action.py

```diff
@@ -1,6 +1,6 @@
 from logging import getLogger
-from simulation.direction import Direction
-from simulation.event import FailedAttackEvent, FailedMoveEvent, MovedEvent, PerformedAttackEvent, ReceivedAttackEvent
+from direction import Direction
+from event import FailedAttackEvent, FailedMoveEvent, MovedEvent, PerformedAttackEvent, ReceivedAttackEvent
```

Comment:
Please revert these changes, we should be using absolute imports, or proper relative ones.

i.e. the only two correct forms here are:
```
from simulation.direction import Direction
from .direction import Direction
```